compose/asc-utils-fonts.c

```diff
@@ -156,7 +156,8 @@ asc_render_font_screenshots (AscResult *cres,
 			as_image_set_height (img, font_screenshot_sizes[j].height);
 			as_image_set_url (img, img_url);
 
-			as_screenshot_add_image (scr, img);
```

Comment:
You shouldn't delete this line, and also move the screenshot addition outside of the image render loop, right now you are adding the same screenshot many times, which will cause problems.